components/pandadoc/actions/send-document/send-document.mjs

```diff
@@ -0,0 +1,86 @@
+import app from "../../pandadoc.app.mjs";
+import { axios } from "@pipedream/platform";
+
+export default defineComponent({
+  key: "pandadoc-send-document",
+  name: "Send Document",
+  description: "Move a document to sent status and send an optional email. [See the docs here](https://developers.pandadoc.com/reference/send-document)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    app,
+    documentId: {
+      propDefinition: [
+        app,
+        "documentId",
+      ],
+    },
+    subject: {
+      type: "string",
+      label: "Subject",
+      description: "Value that will be used as the email subject.",
+      optional: true,
+    },
+    message: {
+      type: "string",
+      label: "Message",
+      description:
+        "A message which will be sent by email with a link to a document to sign.",
+      optional: true,
+    },
+    silent: {
+      type: "boolean",
+      label: "Silent",
+      description:
+        `Disables sent, viewed, comment and completed email notifications for document recipients and the document sender. By default, notifications emails are sent for specific actions. If set as true, it won't affect "Approve document" email notification sent to the Approver.`,
```

Comment:
```suggestion
        `Disables sent, viewed, comment, and completed email notifications for document recipients and the document sender. By default, notification emails are sent for specific actions. If set to true, it won't affect "Approve document" email notification sent to the approver.`
```

Reply:
Added a comma after 'comment' for clarity (Google's Developer docs style guide: Lists), Changed 'emails' to 'email' for consistency (Google's Developer docs style guide: Consistency), Changed 'as true' to 'to true' for clarity (Google's Developer docs style guide: Clarity), Changed 'Approver' to 'approver' for consistency (Google's Developer docs style guide: Capitalization)